### ISSUE TITLE
Can't Delete from OneDrive (and probably any Cloud Folders)

### ISSUE THREAD

**Issue by Maghsad** [jurplel/qView#637]
**Environment:**
- OS: [e.g. macOS 12.7]
- qView version: [e.g. 6.1]

**Information:**
First, thank you for developing this useful app.

Unfortunately, qView cannot delete images in OneDrive:

<img src="https://i.ibb.co/KVwPgvc/SCR-20230927-gtdh.png"/>

It seems this issue may extend to any Cloud Storage that could natively be accessed by Finder (i.e., anything with a physical address like /Users/USERNAME/Library/CloudStorage/CLOUD_FOLDER/IMAGE) since upon attempting to delete, qView tries to move the file to Trash:

<img src="https://i.ibb.co/WDZgLfK/SCR-20230927-gsxk.png"/>

It seems that the issue could be quickly patched if deleting files (at least those located in /Users/USERNAME/Library/CloudStorage/*) be handled by Finder (or an approach similar to what Finder uses is applied):

<img src="https://i.ibb.co/GsnFLLR/SCR-20230927-gyln.png"/>

Thank you again, and looking forward to seeing this bug isfixed.

**Comment by iamvanja** [comment]
Same issue on a network drive (SMB, Mac Ventura).

**Comment by jdpurcell** [comment]
Can't reproduce this with OneDrive, but I do indeed get the same error on a network share. `QFile::moveToTrash` is failing. I tried with `QFile::remove` and it worked, so this would be related to #497.

**Comment by jurplel** [comment]
Makes sense, i'll try to merge the delete action soon and that should fix the issue

**Comment by simonbcn** [comment]
Same problem in MacOS 14 on a SMB folder.
I want to take this opportunity to congratulate the developer of this app. I have tried several image viewers and this is the most complete, fast and efficient I found for Mac.

**Comment by arieller** [comment]
Same issue on files on WSL in windows

**Comment by vassudanagunta** [comment]
I believe the underlying cause is #497. Please upvote that issue.

**Comment by jurplel** [comment]
perma-delete was merged, but is that a good solution? I think we might want to investigate improving the error message to suggest that one may need to use permadelete in the onedrive case.
